t/t-pointer.sh

```diff
@@ -330,8 +334,79 @@ begin_test "pointer --check (with invalid pointer)"
 
   echo "not-a-pointer" > bad.ptr
 
-  ! git lfs pointer --check --file bad.ptr
-  ! git lfs pointer --check --stdin < bad.ptr
+  git lfs pointer --check --file bad.ptr && exit 1
+  git lfs pointer --check --stdin < bad.ptr && exit 1
+  git lfs pointer --check --file bad.ptr && exit 1
+  git lfs pointer --check --stdin < bad.ptr && exit 1
```

Comment:
I think these two lines might now be duplicates of the above two lines, and could be dropped.